distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -253,7 +253,7 @@ type Date
              example_subtract = Date.new 2020 - 7.days
     - : Duration -> Date
     - amount = if amount.is_time then Error.throw (Time.Time_Error "Date does not support time intervals") else
-        Date (this . internal_local_date . minus amount.internal_period)
```

Comment:
One more place. I hope I found all the ones - refactoring for Enso language missing ;-) CCing @jdunkerley